### ISSUE TITLE
Subscription Error

### ISSUE THREAD

**Issue by johtooojod** [botallen/repository.botallen#316]
**When i am logging in hotstar through kodi it not showing me any code and pop up appear of subscription Error

**Comment by vikasvisking** [comment]
Getting the same issue as well @johtooojod 
Is it fixed ?

**Comment by Vikassm73** [comment]
Follow below link

https://github.com/Vikassm73/plugin.video.botallen.hotstar

**Comment by virubhai22** [comment]
> Follow below link
> 
> https://github.com/Vikassm73/plugin.video.botallen.hotstar

i think login problem continues, not working properly.

**Comment by dev27530** [comment]
Yes able to install and login success but chennal are not importing into kodi. After manual click you can view the channel icon but can't play.
